java-iam/license-checks.xml

```diff
@@ -5,6 +5,6 @@
 <module name="Checker">
   <module name="RegexpHeader">
     <property name="fileExtensions" value="java"/>
-    <property name="headerFile" value="${checkstyle.header.file}"/>
+    <property name="headerFile" value="./java-iam/java.header"/>
```

Comment:
Is this correct, or can this defer to the `java.header` and `license-checks.xml` configurations in `./gapic-generator-java-pom-parent`?

Reply:
Note: I see this is consistently applied to the other projects, so you may have already asked this question.

Reply:
The thing is, 
`java-iam`,`java-common-protos`,`gapic-generator-java-pom-parent` modules have the same java.header, whereas 
it is different for `api-common-java` and `gax-java`.  So we have three different header files in the monorepo for now.
This setup I've proposed keeps them as it is and makes the release job work perfectly, but it would be great if we could have a single header for all the modules. Not sure how we can achieve that (I don't know the legal side of this).